forEach/challenges-01.test.js

```diff
@@ -0,0 +1,313 @@
+'use strict';
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named pushIntoSecond which takes in an array and uses a for loop
+// to push all of the elements from the initial array into a second array. 
+// 
+// Return the second array.
+// ------------------------------------------------------------------------------------------------
+
+const pushIntoSecond = (arr) => {
+  const second = [];
+  // Solution code here...
+  for(let i = 0; i < arr.length; i++){
+      second.push(arr[i]);
+  }
+  return second;
+}
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 2
+//
+// Write a function named addBonusPoints that takes in an array of raw test scores
+// and adds five bonus points to each score.
+//
+// Use the for...of syntax.
+// 
+// Return an array of scores that have had the bonus points added.
+// ------------------------------------------------------------------------------------------------
+
+const addBonusPoints = (arr) => {
+  // Solution code here...
+  const newArr = [];
+  for (let i of arr){
+      i += 5;
+      newArr.push(i);
+  }
```

Comment:
you can clean this up and combine lines 36 and 37 into one, `newArr.push(i + 5);`